src/js/edu-benefits/5490/config/form.js

```diff
@@ -34,6 +47,10 @@ const formConfig = {
   confirmation: ConfirmationPage,
   title: 'Update your Education Benefits',
   subTitle: 'Form 22-5490',
+  defaultDefinitions: {
+    educationType: educationType.schema,
```

Comment:
Would this make more sense in the individual page schema, or will it show up in another page as well?